normandy/bundler/views.py

```diff
@@ -0,0 +1,13 @@
+from django.http import HttpResponse, HttpResponseBadRequest
+
+from normandy.recipes.models import Recipe
+
+
+def bundle(request):
```

Comment:
If this was a CBV, it could have a `classmethod` like `Bundler.url_for(bundle)`, so you don't have to do things like `urlparams(reverse('normandy.bundler'), hash=bundle.hash, ids=bundle.ids)`. Or rather, so you can do that complex function call in one centralized place.